{{cookiecutter.github_repository}}/provisioner/roles/postgresql/tasks/main.yml

```diff
@@ -52,6 +52,7 @@
     password: "{{ pg_password }}"
     role_attr_flags: "CREATEDB,SUPERUSER"
     state: present
+    encrypted: yes
```

Comment:
+1

Reply:
Question: I don't see anything changed in the packages, will this install postgres 9.6 or 10? or this is dependent on the operating system used. In any case, it would be worth while to explore into the options to ensure pg 10 is installed and not something else.

Reply:
Saurabh yes, that is what I was exploring yesterday.

In ansible, we have installed postgres package without pinning the version. When tested on Ubunut 18, Postgres version 10 was installed automatically, while for 16.04, it is installing Postgres 9

So, I'm thinking to have condition while generating ansible to have 9th version pin down specifically for Ubuntu 16 and 10th version pinned down for Ubuntu 18

Although I'm not sure if we should do it this way.

What are your thoughts?

Reply:
Also, the `encrypted: yes` is added because it is required in ansible

Reply:
The postgres version should not depend of version of ubuntu. Either of 16 or 18 it should install 10 if postgres 10 is required.

Reply:
I agree with @saurabh-fueled here.

Reply:
Will this also change our database url in any way ? @sanyam-fueled

Reply:
Yes, we're installing 10 in both cases. The only thing that differs here is to get the keys and generate plist file for packages on different versions. That's it !

@akash-fueled No, it won't change the DB URL in any way at least on our server.

For traivs, it will install Postgres 10 in parallel to Postgres 9 and then ensures that it changes the port and config to Postgres 10 so as to test the code in Postgres 10.